react-wasted/src/Foods.js

```diff
@@ -11,12 +11,26 @@ export class Foods extends Component{
         }
     }
 
-    handleAddToCart = (food) => {
+  /*  handleAddToCart = (food) => {
         console.log('in cart');
-        this.setState(prevState => ({
+        console.log(food.ID);
+        this.setState( (prevState) => ({
             cartItems: prevState.cartItems.concat(food)
         }))
+        
     
+    }*/
+
+    addToCart = (ID) =>{
+
+        const {foods, cartItems} = this.state;
+        const data = foods.filter(food =>{
+            return food.ID === ID
```

Comment:
what does  '===' means? just curious whether there is a purpose for one additional equal sign or is it placed just by accident

Reply:
'===' compares both value and type of the variables